<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Окно настроек</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>triangle.png</normaloff>triangle.png</iconset>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="chooseFileButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="pathFileLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="statusTextEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>231</width>
      <height>261</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Информация о статусе процесса триангуляции</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Parameters">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>311</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры (1 пиксель = 1 метр)</string>
    </property>
    <widget class="QSpinBox" name="minHeightSpin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="specialValueText">
      <string/>
     </property>
     <property name="minimum">
      <number>-1000000</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
    <widget class="QLabel" name="minHeightLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Минимальная
высота</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="stepXSpin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="specialValueText">
      <string/>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="maxHeightLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Максимальная
высота</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="maxHeightSpin">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="specialValueText">
      <string/>
     </property>
     <property name="minimum">
      <number>-1000000</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
    <widget class="QLabel" name="stepXLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Шаг по оси X</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="stepYSpin">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>170</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="specialValueText">
      <string/>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
    <widget class="QLabel" name="stepYLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Шаг по оси Y</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="maxDiscrepancySpin">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="specialValueText">
      <string/>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="maxDiscrepancyLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Максимальная невязка</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="MainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="RunAction"/>
   <addaction name="separator"/>
   <addaction name="ExitAction"/>
  </widget>
  <action name="ExitAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>exit.png</normaloff>exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="RunAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>run.png</normaloff>run.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Запустить</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
